mobify/platform-scaffold
# Add window.Progressive.Messaging.supported

Add the `window.Progressive.Messaging.supported` flag, set in the loader, according to the result of calling the SDK `browserSupportsMessaging`, so that it can be checked by the SDK `isMessagingSupported` utility.

 **JIRA**: [MSG-532](https://mobify.atlassian.net/browse/MSG-532)
 **Linked PRs**: [SDK #855](https://github.com/mobify/progressive-web-sdk/pull/855)

## Changes
- In `web/app/loader.js`:
  - Set `MESSAGING_SUPPORTED` by calling `browserSupportsMessaging`
  - Set `window.Progressive.Messaging.supported` from `MESSAGING_SUPPORTED`
  - Check `MESSAGING_SUPPORTED` in `setupMessagingClient`

## Todos

## How to test-drive this PR
- Run the tests
- Using the matching SDK branch, preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Verify that `window.Progressive.Messaging.supported` is set true for Chrome > 46
